client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/style.scss

```diff
@@ -98,3 +98,15 @@
 		background-position-x: 0, 100%;
 	}
 }
+
+
+.is-section-stepper .newsletter.processing {
```

Comment:
Note to self: replicate how videopress applies custom CSS class as a wrapper to use as theming differentiator:

`.is-videopress-stepper.is-processing-step`